modules/cnf-du-configuring-workload-partitioning.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-configuring-workload-partitioning_{context}"]
+
+= Configuring workload partitioning
+
+The following procedure outlines a high level, end to end workflow that deploys a cluster with workload partitioning enabled and pods that are correctly scheduled to run on the management CPU pool.
```

Comment:
For consistency use partition rather than pool. "that are correctly assigned to run on the management CPU partition"

Reply:
Change made.